gradle.properties

```diff
@@ -0,0 +1 @@
+android.useAndroidX=true
```

Comment:
Do we need this?  I don't think we plan to use any `androidx` or `android.support` libs, plus we'd have to move to targetsdk 28: https://developer.android.com/jetpack/androidx#using_androidx_libraries_in_your_project

Reply:
It doesn't compile without this, but I can try to see what's pulling in the androidx stuff.

Reply:
It's a slight concern, but as long as it doesn't affect our build for older devices I guess let's not worry too much.

Reply:
Oh, it looks like we were already using Espresso which includes AndroidX in its [setup instructions](https://developer.android.com/training/testing/espresso/setup#add-espresso-dependencies). That results in some dependencies in the tests like this:
```
androidTestImplementation - Implementation only dependencies for 'androidTest' sources. (n)
\--- androidx.test.espresso:espresso-core:3.1.1 (n)
debugAndroidTestCompileClasspath - Resolved configuration for compilation for variant: debugAndroidTest
+--- androidx.test.espresso:espresso-core:3.1.1
|    +--- androidx.test:runner:1.1.1
|    \--- androidx.test.espresso:espresso-idling-resource:3.1.1
+--- androidx.test.espresso:espresso-core:{strictly 3.1.1} -> 3.1.1 (c)
+--- androidx.test:runner:{strictly 1.1.1} -> 1.1.1 (c)
+--- androidx.test.espresso:espresso-idling-resource:{strictly 3.1.1} -> 3.1.1 (c)
+--- androidx.annotation:annotation:{strictly 1.0.0} -> 1.0.0 (c)
\--- androidx.test:monitor:{strictly 1.1.1} -> 1.1.1 (c)

```

Reply:
Oops, my bad... I only searched for imports in Java files.  Thanks.